packages/ftp/test/list.test.js

```diff
@@ -1,57 +1,22 @@
 import { deepStrictEqual, rejects, strictEqual } from 'assert'
-import { readFileSync } from 'fs'
-import { resolve } from 'path'
-import * as url from 'url'
-import getStream from 'get-stream'
+import { getStreamAsArray } from 'get-stream'
 import list from '../list.js'
 import FtpServer from './support/FtpServer.js'
 import { withServer } from './support/server.js'
-import SftpServer from './support/SftpServer.js'
-
-const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
+import serverConfigurations from './support/serverConfigurations.js'
 
 describe('list', () => {
   it('is a function', () => {
     strictEqual(typeof list, 'function')
   })
 
-  ;[
-    [
-      'on a FTP server with anonymous user',
-      () => new FtpServer(),
-      {},
-    ],
-    [
-      'on a FTP server with username/password',
-      () => new FtpServer({ user: 'test', password: '1234' }),
-      {},
-    ],
-    [
-      'on a SFTP server with anonymous user',
-      () => new SftpServer(),
-      {},
-    ],
-    [
-      'on a SFTP server with username/password',
-      () => new SftpServer({ user: 'test', password: '1234' }),
-      {},
-    ],
-    [
-      'on a SFTP server with private key',
-      () => new SftpServer({ user: 'test', password: '1234' }),
-      { password: undefined, privateKey: readFileSync(resolve(__dirname, 'support/test.key')) },
-    ], [
-      'on a SFTP server with private key specified as a file',
-      () => new SftpServer({ user: 'test', password: '1234' }),
-      { password: undefined, privateKey: resolve(__dirname, 'support/test.key') },
-    ],
-  ].forEach(([label, serverFactory, additionalOptions]) => {
+  serverConfigurations.forEach(([label, serverFactory, additionalOptions]) => {
     it(`lists files from the given directory ${label}`, async () => {
       await withServer(serverFactory, async server => {
         const options = { ...server.options, ...additionalOptions }
 
         const stream = await list({ pathname: 'data', ...options })
-        const filenames = await getStream.array(stream)
```

Comment:
I needed to change `getStream.array` to `getStreamAsArray` so that it's working ; it was saying that `array` is not a function.
It seems that they did some breaking changes in `get-stream`.

Reply:
Yes, there was that change to `get-stream` along the way